src/CSnakes.Runtime/PythonLogger.cs

```diff
@@ -109,10 +117,11 @@ private static Task StartRecordListener(PyObject handler, ILogger logger)
     {
         using PyObject getRecords = handler.GetAttr("get_records");
         using PyObject getRecordsResult = getRecords.Call();
-        IGeneratorIterator<(long, string, ExceptionInfo?), PyObject, PyObject> generator =
-            getRecordsResult.ImportAs<IGeneratorIterator<(long, string, ExceptionInfo?), PyObject, PyObject>,
-                                                         PyObjectImporters.Generator<(long, string, ExceptionInfo?), PyObject, PyObject,
-                                                                                     PyObjectImporters.Tuple<long, string, ExceptionInfo?,
+        IGeneratorIterator<(long, long, string, ExceptionInfo?), PyObject, PyObject> generator =
+            getRecordsResult.ImportAs<IGeneratorIterator<(long, long, string, ExceptionInfo?), PyObject, PyObject>,
+                                                         PyObjectImporters.Generator<(long, long, string, ExceptionInfo?), PyObject, PyObject,
```

Comment:
Is this change from the other importer PRs?

Reply:
No, this is just adding extending the tuple with an additional `long` for the drop count, going from `(long, string, ExceptionInfo?)` to `(long, long, string, ExceptionInfo?)` Unfortunately, given the number of importers and their type arguments involved (exacerbated by the optional `ExceptionInfo` tuple being nested in another tuple), I had to keep things aligned with formatting/indentation for sanity's sake and so the delta here appears bigger than it is. The good news is that the compiler statically guarantees that this setup is correct otherwise it'll reject the compilation.